ciscripts/buildpending.sh

```diff
@@ -2,5 +2,6 @@
 curl -XPOST -H "Authorization: token $GITHUB_SECRET" https://api.github.com/repos/steemit/steem/statuses/$(git rev-parse HEAD) -d "{
```

Comment:
thanks for figuring out all this arcane bullshit @jredbeard - super glad i don't have to. :D